tests/utils.go

```diff
@@ -320,16 +331,24 @@ func (w *ObjectEventWatcher) Watch(ctx context.Context, processFunc ProcessFunc,
 
 	if w.failOnWarnings {
 		f = func(event *k8sv1.Event) bool {
+			isIgnored := false
```

Comment:
Change into `shouldNotIgnoreThisWarning` to have some context without know that we are into `failOnWarnings`